### DISCUSSION TITLE
Layer doesn't allow string that uses a package wildcard twice

### DISCUSSION THREAD

**Discussion by xplayerCZ** [LemonAppDev/konsist#765] [Report a Problem]
### Tell us what version of Konsist you use.

0.13.0

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

When constructing a Layer with string like "..domain..", inside the assertArchitecture, it doesn't pass the internal Regex. Thus when I want the test to be used across multiple packages, it doesn't let me.

For example: `val domain = Layer("domain", "..domain..")`

### Actual behaviour

The test crashed on error

`com.lemonappdev.konsist.core.exception.KoPreconditionFailedException: Layer domain must be defined by package ending with '..'. Now: ..domain.. .`

### Expected behaviour

I would expect the Regex to allow strings using multipl ".." and match all of the packages that match the format.

**Comment by igorwojda** [comment]
I can confirm this is a bug. 
I will prioretyze this for the upcoming release.

**Comment by Mohamed-Mabrouki** [comment]
i have faced the same problem so tried to find solution that works on all the project without specifying the feature name:
	@Test
```
	fun `checking that data layer does not depend on ui layer`() {
		Konsist.scopeFromProject() // project scope
			.packages //all project packages
			.containingFiles //all files in all the packages
			.withPackage("..data..") //filtering to only leave the files that are in the data package
			.assertFalse { file ->
				file.hasImport { imports ->
					imports.hasNameContaining(".ui.")
				}            //asserting that those files do not depend on ui layer
			}
	`}`
```
and it seems to be working correctly

**Reply by nataliapeterwas** [reply]
A new version v0.14.0 has been released. It contains the fix.
See [v0.14.0 release](https://github.com/LemonAppDev/konsist/releases/tag/v0.14.0)

**Comment by igorwojda** [comment]
Fix is in progress https://lemonappdev.atlassian.net/browse/KON-577.
It will added to the next release (coming in few days).

**Comment by nataliapeterwas** [comment]
A new version v0.14.0 has been released. It contains the fix.
See [v0.14.0 release](https://github.com/LemonAppDev/konsist/releases/tag/v0.14.0)
